frontend/src/container/ListOfDashboard/index.tsx

```diff
@@ -271,19 +281,21 @@ function ListOfAllDashboard(): JSX.Element {
 					uploadedGrafana={uploadedGrafana}
 					onModalHandler={(): void => onModalHandler(false)}
 				/>
-				<Table
-					pagination={{
-						pageSize: 9,
-						defaultPageSize: 9,
-					}}
-					showHeader
-					bordered
-					sticky
-					loading={loading}
-					columns={columns}
-					dataSource={data}
-					showSorterTooltip
-				/>
+				<ResizeTableWrapper columns={columns}>
+					<Table
+						pagination={{
```

Comment:
as this object is constant please move this one config

Reply:
@palashgdev Pagination calculation is not a constant, its different for across component.